composer/loggers/tensorboard_logger.py

```diff
@@ -136,6 +139,36 @@ def eval_end(self, state: State, logger: Logger) -> None:
     def fit_end(self, state: State, logger: Logger) -> None:
         self._flush(logger)
 
+    def log_images(
+        self,
+        images: Union[np.ndarray, torch.Tensor, Sequence[Union[np.ndarray, torch.Tensor]]],
+        name: str = 'Images',
+        channels_last: bool = False,
+        step: Optional[int] = None,
+        masks: Optional[Dict[str, Union[np.ndarray, torch.Tensor, Sequence[Union[np.ndarray, torch.Tensor]]]]] = None,
+        mask_class_labels: Optional[Dict[int, str]] = None,
+        use_table: bool = False,
+    ):
+        def _to_numpy_float16(t):
+            if isinstance(t, torch.Tensor):
+                return t.to(torch.float16).cpu().numpy()
+            if isinstance(t, list):
+                return np.array([_to_numpy_float16(image) for image in t])
+            return t
```

Comment:
Can we make this a helper function that's outside the class (equivalent to `_convert_to_wandb_image`)?

Reply:
Done :)